onnxruntime/core/graph/contrib_ops/bert_defs.cc

```diff
@@ -199,6 +199,10 @@ ONNX_MS_OPERATOR_SET_SCHEMA(
               "(2, batch_size, num_heads, max_sequence_length, head_size)",
               AttributeProto::INT,
               OPTIONAL_VALUE)
+        .Attr("do_rotary",
+              "Whether use rotary position embedding. Default value is 0.",
+              AttributeProto::INT,
+              static_cast<int64_t>(0))
```

Comment:
please add same attribute for QAttention

Reply:
added